x-pack/plugins/maps/ui_tests/fixtures/page_objects/gis_page.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ScoutPage} from '@kbn/scout';
+
+export class GisPage {
+  constructor(private readonly page: ScoutPage) {}
+
+  async fullScreenModeMenuItemExists() {
+    await this.page.testSubj.locator('mapsFullScreenMode').waitFor({ state: 'visible' });
+  }
```

Comment:
if the goal is to make assertion, you need to use `expect` so it fails with clear message:

```ts
      await expect(
        this.page.testSubj.locator('mapsFullScreenMode'),
        'Full screen mode should be active'
      ).toBeVisible();
```

I believe it is better to put it as in test instead of wrapping inside Page Objects. This way all assertions with `expect` are in test file and we can analyse tests based on assertions. "tests without assertions aren’t testing much"